src/main/java/com/sopt/collaboration/service/BookService.java

```diff
@@ -56,6 +58,25 @@ public BookSearchResponseDto searchBooks(String keyword) {
                 bannerImageUrl
         );
     }
+
+    public BookDetailResponseDto getBookDetail(Long bookId) {
+
+        Book book = bookRepository.findById(bookId)
+                .orElseThrow(() -> new BaseException(ErrorCode.BOOK_NOT_FOUND));
+
+        String bookImageUrl = s3Service.generatePresignedUrlOrNull(book.getBookImageKey());
+
+        List<ReviewSummaryDto> reviewDtos = book.getReviews().stream()
```

Comment:
Book을 조회하고 리뷰를 getReviews로 한번 더 조회하는건 @OneToMany LAZY 로딩으로 인한 N+1이 발생해요!

```suggestion
  // BookRepository.java                                                                                                                                                                                 
  @Query("SELECT b FROM Book b LEFT JOIN FETCH b.reviews WHERE b.bookId = :bookId")                                                                                                                      
  Optional<Book> findByIdWithReviews(@Param("bookId") Long bookId);

  // BookService.java                                                                                                                                                                                    
  public BookDetailResponseDto getBookDetail(Long bookId) {
      Book book = bookRepository.findByIdWithReviews(bookId)
              .orElseThrow(() -> new BaseException(ErrorCode.BOOK_NOT_FOUND));
```

이런 식으로 한번에 조회하는게 좋아요